### PR TITLE
add highlight task

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

File: tasks/highlight.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+const through = require('through2');
+
+function highlight() {
+  return gulp.src(config.src.www_code_pages)
+    .pipe(through.obj(function(file, enc, cb) {
+      if (file.isNull()) {
+        cb(null, file);
+        return;
+      }
+      file.contents = new Buffer(highlighter(file.contents.toString(),
```
Comment (camelburrito) [inline tasks/highlight.js]: did you explore if it was possible to replace the code in a div with a highlight? (if this is too hard we can work around it by using different source and dest folders :) )

File: tasks/config.js [hunk 2]
```diff
@@ -27,6 +28,7 @@ module.exports = {
     default: 'dist',
     templates: 'dist',
     www_pages: 'dist',
+    www_code_pages: 'dist/www/code',
```
Comment (camelburrito) [inline tasks/config.js]: are we pushing it in the same file? we should put it in www/hl/<same-file-name> ?

File: tasks/highlight.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+          'html'));
+      this.push(file);
+    }))
+    .pipe(gulp.dest(config.dest.www_code_pages));
```
Comment (camelburrito) [inline tasks/highlight.js]: how do we add the css file needed for the highlight ?

File: tasks/highlight.js (hunk 1 above)
Comment (erwinmombay) [inline tasks/highlight.js]: not sure what you mean by code in a div, can you post a snippet?

File: tasks/config.js (hunk 2 above)
Comment (erwinmombay) [inline tasks/config.js]: no idea, feel free to change after the fact or lmk what the path should be

File: tasks/highlight.js (hunk 3 above)
Comment (erwinmombay) [inline tasks/highlight.js]: lets download https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/styles/default.min.css

Comment (camelburrito) [inline tasks/highlight.js]: @erwinmombay can you also put where you got this link from ?

Comment (erwinmombay) [inline tasks/highlight.js]: from https://highlightjs.org/download/

**Pull request merged**